.zuul.d/collect-logs.yml

```diff
@@ -15,13 +15,29 @@
             chdir: "{{ ansible_user_dir }}/zuul-output/logs"
             cmd: cp /tmp/report.html .
 
+    - name: Check if we get ci-framework basedir
+      register: cifmw_state
+      ansible.builtin.stat:
+        path: "{{ ansible_user_dir }}/ci-framework"
+
+    - name: Copy ci-framework interesting files
+      when:
+        - cifmw_state.stat.exists | bool
+      ansible.builtin.shell:
+        chdir: "{{ ansible_user_dir }}/zuul-output/logs/"
+        cmd: |
+          mkdir ci-framework; pushd ci-framework
```

Comment:
Can we convert directory creation to ansible and then call shell module to copy log files? We can use block for that if we use ansible.

Reply:
right - #88 will fix this.